_drafts/diving-into-jtag-part5.md

```diff
@@ -8,12 +7,11 @@ tags: [arm, cortex-m, mcu, debugging, debugger]
 
 <!-- excerpt start -->
 
-The main examples of JTAG usage, such as debugging and testing of boards in production,
-we have considered in the previous parts. And for firmware/embedded developers only the
-first example - debugging - is the most useful. As far as I know, we almost never encounter
-the second option, although it can be very useful too. And in this article I want to look
-at two examples of JTAG Boundary Scan which I think can be very useful in everyday work of
-a firmware/embedded developer: Bring-up and Revers Enginnering.
+In previous articles we have considered the main examples of JTAG usage, such as debugging
+and testing boards in production. For firmware/embedded developers, the
+first example - debugging - is the most useful. In this article, I want to look
+at two examples of JTAG Boundary Scan, which can be very useful in the everyday work of
```

Comment:
I like the rework but I think we could clarify the language some more. How about:
```suggestion
In previous articles, we have considered the primary uses of JTAG, including debugging
and testing boards in production. For firmware/embedded developers, the
first - debugging - is the most common. In this article, I want to look
at two uses of JTAG Boundary Scan, which are also common tasks for
a firmware/embedded developer: board bring-up and reverse engineering.
```